include/os/linux/spl/sys/file.h

```diff
@@ -26,7 +26,6 @@
 
 #define	FIGNORECASE		0x00080000
 #define	FKIOCTL			0x80000000
-#define	ED_CASE_CONFLICT	0x10
```

Comment:
Whoops, looks like this one is still used.

Reply:
I just put that back. Thanks for catching it so quickly.